docs/en/Calling-Alexa-Service-APIs.rst

```diff
@@ -0,0 +1,590 @@
+
+.. tabs::
+
+  .. code-tab:: javascript
+
+    function callDirectiveService(handlerInput, date) {
+      const requestEnvelope = handlerInput.requestEnvelope;
+      const directiveServiceClient = handlerInput.serviceClientFactory.getDirectiveServiceClient();
+
+      const requestId = requestEnvelope.request.requestId;
+      const directive = {
+        header: {
+          requestId,
+        },
+        directive: {
+          type: 'VoicePlayer.Speak',
+          speech: `$Please wait while I look up information about ${date}...`,
+        },
+      };
+
+      return directiveServiceClient.enqueue(directive);
+    }
+
+  .. code-tab:: typescript
+
+    import { HandlerInput } from 'ask-sdk-core';
+    import { services } from 'ask-sdk-model';
+    import SendDirectiveRequest = services.directive.SendDirectiveRequest;
+
+    function callDirectiveService(handlerInput : HandlerInput, date : string) : Promise<void> {
+      const requestEnvelope = handlerInput.requestEnvelope;
+      const directiveServiceClient = handlerInput.serviceClientFactory.getDirectiveServiceClient();
+
+      const requestId = requestEnvelope.request.requestId;
+
+      const directive : SendDirectiveRequest = {
+          header: {
+              requestId,
+          },
+          directive: {
+              type: 'VoicePlayer.Speak',
+              speech: `$Please wait while I look up information about ${date}...`,
+          },
+      };
+
+      return directiveServiceClient.enqueue(directive);
+    }
+
+ListManagementServiceClient
+===========================
+
+``ListManagementServiceClient`` can be used to access the `List Management API <https://developer.amazon.com/docs/custom-skills/access-the-alexa-shopping-and-to-do-lists.html#list-management-quick-reference>`_ in order to read or modify both the Alexa default lists and any custom lists customer may have.
+
+Type Definition
+---------------
+
+.. code-block:: typescript
+
+  class ListManagementServiceClient {
+    getListsMetadata(): Promise<services.listManagement.AlexaListsMetadata>;
+    getList(listId: string, status: string): Promise<services.listManagement.AlexaList>;
+    getListItem(listId: string, itemId: string): Promise<services.listManagement.AlexaListItem>;
+    createList(createListRequest: services.listManagement.CreateListRequest): Promise<services.listManagement.AlexaListMetadata>;
+    createListItem(listId: string, createListItemRequest: services.listManagement.CreateListItemRequest): Promise<services.listManagement.AlexaListItem>;
+    updateList(listId: string, updateListRequest: services.listManagement.UpdateListRequest): Promise<services.listManagement.AlexaListMetadata>;
+    updateListItem(listId: string, itemId: string, updateListItemRequest: services.listManagement.UpdateListItemRequest): Promise<services.listManagement.AlexaListItem>;
+    deleteList(listId: string): Promise<void>;
+    deleteListItem(listId: string, itemId: string): Promise<void>;
+  }
+
```

Comment:
In case we are not providing code samples, can we provide links to the code examples in cookbook here?